js/parts/ColumnSeries.js

```diff
@@ -577,7 +577,9 @@ seriesType('column', 'line',
         Series.prototype.translate.apply(series);
         // Record the new values
         series.points.forEach(function (point) {
-            var yBottom = pick(point.yBottom, translatedThreshold), safeDistance = 999 + Math.abs(yBottom), pointWidth = seriesPointWidth, 
+            var yBottom = pick(point.yBottom, translatedThreshold, 0 // Handle the situation when the plot area is not
+            // visible because of multiple legends.
+            ), safeDistance = 999 + Math.abs(yBottom), pointWidth = seriesPointWidth, 
```

Comment:
What is the reason of using 999 ? Why i.e 1294?

Reply:
That line of code is not written by me. I've just added another argument to `pick` function.